content/webapp/components/Exhibition/Exhibition.tsx

```diff
@@ -259,6 +270,156 @@ const Exhibition: FunctionComponent<Props> = ({
     []
   );
 
+  const visualStoryLink = accessResourceLinks.find(
+    link => link.type === 'visual-story'
+  );
+
+  // Theoretically, there could be multiple ExhibitionTexts and ExhibitionHighlightTours
+  // attached to an exhibition, but in reality there is only one, so we just take the first
+  // and create links to them.
+  const exhibitionTextLink = linkResolver(exhibitionTexts[0]);
+  const bslTourLink = linkResolver({
+    ...exhibitionHighlightTours[0],
+    highlightTourType: 'bsl',
+  });
+  const audioTourLink = linkResolver({
+    ...exhibitionHighlightTours[0],
+    highlightTourType: 'audio',
+  });
+
+  const accordionContent = [
+    {
+      summary: 'Digital highlights tour',
+      content: (
+        <ul>
+          <li>
+            Find out more about the exhibition with our digital highlights tour,
+            available in short audio clips with audio description and
+            transcripts, or as BSL videos.It can be accessed on your own device,
+            via handheld devices with tactile buttons, or on an iPad which you
+            can borrow
+          </li>
+          <li>
+            <NextLink href={bslTourLink}>Watch BSL video tour</NextLink>
+          </li>
+          <li>
+            <NextLink href={audioTourLink}>
+              Listen to audio tour with audio description
+            </NextLink>
+          </li>
+        </ul>
+      ),
+    },
+    {
+      summary: 'BSL, transcripts and induction loops',
+      content: (
+        <ul>
+          <li>Audiovisual content is available in BSL in the gallery</li>
+          <li>
+            Live BSL tours are available. See our exhibition events above for
+            more information or contact us in advance to request a tour
+          </li>
+          <li>
+            <NextLink href={bslTourLink}>
+              Watch BSL videos of the digital highlights tour on your own device
+            </NextLink>
+          </li>
+          <li>
+            <span id="transcript-link-text">
+              Transcripts of all audiovisual content are available
+            </span>{' '}
+            in the gallery and{' '}
+            <NextLink
+              id="transcript-link"
+              aria-labelledby="transcript-link-text transcript-link"
+              href={exhibitionTextLink}
+            >
+              online
+            </NextLink>
```

Comment:
```suggestion
            in the gallery{' '}
            <NextLink
              id="transcript-link"
              aria-labelledby="transcript-link-text transcript-link"
              href={exhibitionTextLink}
            >
             and online
            </NextLink>
```

Reply:
This is [dealt with in another PR](https://github.com/wellcomecollection/wellcomecollection.org/pull/11658/files#diff-e3fe2a365deb2dc15410a0a763f6730d2363fb193e891401c18682705a5adee5R348).

I didn't want the 'and' to be part of the link text. I'm using the aria-labelledby attribute to create the accessible name for the link and want it to read "Transcripts of all audiovisual content are available online"

Reply:
yeah tbf it probably makes sense.